### DISCUSSION TITLE
Why do I have double links for previous/next page in my blog?

### DISCUSSION THREAD

**Discussion by i-do** [datenstrom/community#534] [Ask a question] [answered]
Can't find where I can change shortcuts for core [pagePrevious] [pageNext]. 

Found extension witch do the same thing but it's perfectly lay straight under the core [pagePrevious] [pageNext] so I have double links in my Blog.

I need some advice how can I configure this parameters.

**Comment by schulle4u** [comment, marked as answer]
Do you still have the old links extension installed? It has been renamed to `previousnext`. Check your extensions folder for the `links.php` file and delete it, this should solve the problem. 

Steffen

**Comment by i-do** [comment]
@schulle4u, you're right! 
Thanks a lot!
